src/main/java/Barney.java

```diff
@@ -1,55 +1,89 @@
 import java.util.Scanner;
-import java.util.ArrayList;
 
 public class Barney {
-    private static String LONG_LINE = "_________________________________________________";
-    private static Scanner SCANNER = new Scanner(System.in);
+    private static String LONG_LINE = "____________________________________________________________";
 
     public static void main(String[] args) {
-        welcomeText();
-        chat();
-        endingText();
-    }
-
-    public static void welcomeText() {
-        String text = "Hello, I am Barney <RAWR>, what can I do for you?";
-        System.out.println(text);
+        // Welcome text
+        String welcomeText = "Hello, I am Barney <RAWR>, what can I do for you?";
+        System.out.println(welcomeText);
         System.out.println(LONG_LINE);
-    }
 
-    public static void chat() {
+        Scanner SCANNER = new Scanner(System.in);
         String input;
+
         Boolean isChatting = true;
-        String[] list = new String[100];
+
+        Task[] taskList = new Task[100];
         int listLength = 0;
 
         while (isChatting) {
             System.out.println(">>>");
-            input = SCANNER.nextLine();
+            input = SCANNER.nextLine().trim();
             System.out.println("<<<");
 
-            switch (input) {
+            String[] inputArray = input.split(" ");
+            String keyword = inputArray[0];
+
+            switch (keyword) {
                 case "list":
-                    for (int i = 1; i <= listLength; i++) {
-                        System.out.println(i + ". " + list[i - 1]);
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < listLength; i++) {
+                        System.out.println(String.format("%d. %s", i + 1, taskList[i].toString()));
```

Comment:
Perhaps these 3 lines of code could be indented since they are within the first loop?